src/web/types/datetime-const.ts

```diff
@@ -0,0 +1,54 @@
+export enum HoursConst {
+    ZERO_HOURS = 0,
+    TWELVE_HOURS = 12,
+    ONE_DAY_HOURS = 24,
+    THREE_DAYS_HOURS = 72,
+    ONE_WEEK_HOURS = 168,
+}
+
+export enum SecondsConst {
+    ONE_MINUTE_SECONDS = 60,
+}
+
+export enum MinutesConst {
+    ONE_HOURS_MINUTES = 60,
+}
+
+export enum MsConst {
+    ONE_SECOND_MILLISECONDS = 1000,
```

Comment:
I'm thinking the current format `MsConst.ONE_SECOND_MILLISECONDS` may not be immediately obvious when used.

Would something like `Milliseconds.IN_ONE_SECOND` be clearer?